src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -41,19 +42,7 @@ $active-poster-shadows: 0 10px 20px 0 var(--primary--under-50a);
 
   &__header {
     display: flex;
-    justify-content: space-between;
-  }
-
-  &__posterId {
-    display: inline;
-    text-align: left;
-    font-size: $font-size--sm;
-  }
-
-  &__visiting {
-    display: inline;
-    text-align: right;
-    font-size: $font-size--sm;
+    justify-content: left;
```

Comment:
Better than centered?

Reply:
It's what has been designed. They're all going to get redesigned soon anyway.